library/core/src/num/mod.rs

```diff
@@ -222,6 +224,12 @@ impl u8 {
         *self | (self.is_ascii_uppercase() as u8 * ASCII_CASE_MASK)
     }
 
+    /// Assumes self is ascii
+    #[inline]
+    pub(crate) fn ascii_change_case_unchecked(&self) -> u8 {
+        *self ^ ASCII_CASE_MASK
```

Comment:
Why can't we put the `const` here rather than at the top? Isn't it easier to find here?